srcpkgs/shotgun/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'shotgun'
+pkgname=shotgun
+version=2.2.1
```

Comment:
2.3.1 seems to be the latest one upstream

Reply:
It has been bumped to 2.3.1 in master, but there has been no release with this version yet. It might be a version number mismatch, but this package works with the release version. See https://github.com/neXromancers/shotgun/releases.

Reply:
I think they just do tags and not releases now. Most other distributions package 2.3.1, so we should do so too: https://repology.org/project/shotgun/versions

Reply:
Ok, I'll update the package version.

Reply:
Should be ready.

Reply:
commit message should also be adjusted

Reply:
I forgot about that ... fixed.